src/tokio.rs

```diff
@@ -0,0 +1,40 @@
+use tokio::process::Command;
+use tokio_crate as tokio;
+
+use crate::*;
```

Comment:
Style nit: I prefer not to have wildcard imports, please import the types you need explicitly.